test/testenv/appframework_utils.go

```diff
@@ -311,3 +325,300 @@ func WaitforPhaseChange(deployment *Deployment, testenvInstance *TestEnv, name s
+				VerifyAppsCopied(deployment, testenvInstance, testenvInstance.GetName(), allPodNames, appList, false, false)
+				allPodNames = append(allPodNames, GeneratePodNameSlice(SearchHeadPod, deployment.GetName(), appSource.CrReplicas, false, 1)...)
+				testenvInstance.Log.Info(fmt.Sprintf("Verify %s apps with 'cluster' scope are copied to /etc/apps/ on Search Heads", appVersion))
+				VerifyAppsCopied(deployment, testenvInstance, testenvInstance.GetName(), allPodNames, appList, true, true)
+			}
+			if appSource.CrKind == "Standalone" {
+				testenvInstance.Log.Info(fmt.Sprintf("Verify %s apps with 'local' scope are copied to /etc/apps/ on Standalone", appVersion))
+				standalonePodName := fmt.Sprintf(StandalonePod, appSource.CrName, 0)
+				if scaling == "up" {
+					podNames := []string{standalonePodName}
+					podNames = append(podNames, fmt.Sprintf(StandalonePod, deployment.GetName(), 1))
+					VerifyAppsCopied(deployment, testenvInstance, testenvInstance.GetName(), podNames, appList, true, false)
+				} else {
+					VerifyAppsCopied(deployment, testenvInstance, testenvInstance.GetName(), []string{standalonePodName}, appList, true, false)
+				}
+			}
+			if appSource.CrKind == "MonitoringConsole" {
+				testenvInstance.Log.Info(fmt.Sprintf("Verify %s apps with 'local' scope are copied to /etc/apps/ on Monitoring Console", appVersion))
+				mcPodName := fmt.Sprintf(MonitoringConsolePod, appSource.CrName, 0)
+				VerifyAppsCopied(deployment, testenvInstance, testenvInstance.GetName(), []string{mcPodName}, appList, true, false)
+			}
+		}
+	}
+
+	// Verify no pods reset by checking the pod age
+	testenvInstance.Log.Info("Verify no pods reset by checking the pod age")
+	if scaling == "up" || scaling == "down" {
+		for _, appSource := range appSource {
+			if appSource.CrKind == "Standalone" {
+				// Excluding MC pod from list of pods to verify at it will reset after scaling
+				mcPodName := fmt.Sprintf(MonitoringConsolePod, appSource.CrName, 0)
+				VerifyNoPodReset(deployment, testenvInstance, testenvInstance.GetName(), splunkPodAge, []string{mcPodName})
+			}
+			if appSource.CrKind == "SearchHeadCluster" {
+				// Excluding SHC pods from list of pods to verify at they will reset after scaling
+				shcPodNames := []string{fmt.Sprintf(DeployerPod, deployment.GetName())}
+				shcPodNames = append(shcPodNames, GeneratePodNameSlice(SearchHeadPod, deployment.GetName(), appSource.CrReplicas, false, 1)...)
+				VerifyNoPodReset(deployment, testenvInstance, testenvInstance.GetName(), splunkPodAge, shcPodNames)
+			}
+		}
+	} else {
+		VerifyNoPodReset(deployment, testenvInstance, testenvInstance.GetName(), splunkPodAge, nil)
+	}
+
+	// Verify apps are installed at correct location on the pods
+	var checkupdated bool
+	if appVersion == "V1" {
+		checkupdated = false
+	} else if appVersion == "V2" {
+		checkupdated = true
+	}
+	if appFileList2ndScopeCluster != nil && appList2ndScopeCluster != nil {
```

Comment:
Similar comments to above around `appFileList2ndScopeCluster`. I dont understand the usecase for `appFileList2ndScopeCluster`